opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/__init__.py

```diff
@@ -384,8 +384,10 @@ def force_flush(self, timeout_millis: float = 10_000) -> bool:
             current_ts = _time_ns()
             try:
                 if current_ts >= deadline_ns:
-                    raise Exception("Timed out while flushing metric readers")
-                metric_reader.collect(
+                    raise TimeoutError(
```

Comment:
I was going to use this before but decided against it, check the docs https://docs.python.org/3/library/exceptions.html#TimeoutError:

> Raised when a **system function timed out at the system level**. Corresponds to errno ETIMEDOUT.

Reply:
Fixed